### PR TITLE
card display component code

### PR DESCRIPTION
**Pull request by shongekar**
card component code

### CODE REVIEW

File: src/app/common/components/card/card.component.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
```
Comment (sachkanch2010) [inline src/app/common/components/card/card.component.ts]: remove OnInit

File: src/app/common/components/card/card.component.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-  selector: 'app-card',
+  selector: 'db-app-card',
   templateUrl: './card.component.html',
   styleUrls: ['./card.component.css']
 })
-export class CardComponent implements OnInit {
+export class CardComponent {
 
-  constructor() { }
+  @Input ('rank') public rank: string = 'A';   // card value
```
Comment (sachkanch2010) [inline src/app/common/components/card/card.component.ts]: Remove default value

File: src/app/common/components/card/card.component.ts [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+  @Input ('suit') public suit = 'spades'; // card symbols
```
Comment (sachkanch2010) [inline src/app/common/components/card/card.component.ts]: add space and change from symbols to symbol

File: src/app/common/components/card/card.component.ts [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
 
-  ngOnInit() {
+  public getColor(symbol: string): string {
+    if  (symbol === 'hearts' || symbol === 'diamonds') {
+    return 'red';
```
Comment (sachkanch2010) [inline src/app/common/components/card/card.component.ts]: indent

File: src/app/common/components/card/card.component.ts (hunk 2 above)
Comment (sachkanch2010) [inline src/app/common/components/card/card.component.ts]: Convert to Object

File: src/app/common/components/card/card.component.ts (hunk 3 above)
Comment (shongekar) [reply src/app/common/components/card/card.component.ts]: taken care

File: src/app/common/components/card/card.component.ts (hunk 4 above)
Comment (shongekar) [reply src/app/common/components/card/card.component.ts]: taken care

File: src/app/common/components/card/card.component.ts (hunk 2 above)
Comment (shongekar) [reply src/app/common/components/card/card.component.ts]: Taken care

Comment (shongekar) [reply src/app/common/components/card/card.component.ts]: taken care

**Pull request merged**